avcastoldi/galvanize-rest-api
# #1 As a User, I want to manage products using a REST API

This branch contains the following functionality:
* A blank new Rails 6.0.3 API application

## Documentation
* Basic gems to support the development (`rubocop`, `guard`, `annotate`) were added.
* Instructions about the problem and how to install the application on README.md

## How to test
1. Install the required gems
    ```bash
    bundle install
    bundle exec rails db:setup
    ```
2. Verify if the source code adheres to the coding standards
    ```bash
    bundle exec rubocop
    ```
    **Expected result:**
![image](https://user-images.githubusercontent.com/31883/115158876-440fb600-a045-11eb-85bc-b07cd576e89c.png)

3. Run the test suite
    ```bash
    bundle exec rails test
    ```
   **Expected Result**
![image](https://user-images.githubusercontent.com/31883/115158961-a8cb1080-a045-11eb-8e85-4841bcf7fff8.png)

4. Launch the Application and visit http://localhost:3000
    ```bash
    bundle exec rails server
    ```
    **Expected Result**
    
![image](https://user-images.githubusercontent.com/31883/115159012-ed56ac00-a045-11eb-8ac1-56aa55532885.png)
